typescript definition in package.json causes tsd install to misbehave?

We've just updated to `2.5.0-dev3` and _I think_ got bitten by [this change](https://github.com/palantir/tslint/commit/c510503fcdf87efe60bfba0e0b63e422b4a86f66#diff-b9cfc7f2cdf78a7f4b91a753d10865a2)

Our `tsd.d.ts` file does not contain a reference to `typescriptServices.d.ts` and hence the namespace `ts` is unavailable. Correspondingly we see our build fail:

```
node_modules/tslint/lib/tslint.d.ts(222,31): error TS2503: Cannot find namespace 'ts'
```

But then I was confused as to how we were even referencing `tslint.d.ts` in the first place. 

From what I can tell, this move to add the `typescript` block to `package.json` triggers `tsd install` to automatically add a reference into an app's `tsd.d.ts` (I might have this wrong). Can't be removed... because `tsd install` (which is what we obviously run in our CI) reinstates it.

Any thoughts on how we go about fixing this?

I'm not keen on referencing `typescriptServices.d.ts` because we don't need it (right now)

Thanks

--- a/package.json
+++ b/package.json
@@ -6,9 +6,6 @@
     "tslint": "./bin/tslint"
   },
   "main": "./lib/tslint",
-  "typescript": {
-    "definition": "lib/tslint.d.ts"
-  },
   "repository": {
     "type": "git",
     "url": "https://github.com/palantir/tslint.git"

